docs/authoring-recipes/recipe-development-environment.md

```diff
@@ -108,11 +112,7 @@ dependencies {
 
 ```xml title="pom.xml"
 <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
Those options will be explained in the "Set Language Level and Bytecode Level" section. This portion of the documentation should focus on dependencies primarily and not compiler options, as indicated by the subtitle.